lib/vagrant-vbguest/installers/linux.rb

```diff
@@ -79,11 +79,11 @@ def running?(opts=nil, &block)
       #                  available on the guest, or `nil` if none installed.
       def guest_version(reload = false)
         return @guest_version if @guest_version && !reload
-        driver_version = super
+        driver_version = super.to_s.match(/^(\d+\.\d+.\d+)/)
```

Comment:
Small hint: You can use rubies "Element Reference" feature to directly get the matched substring instead of an match array:

```ruby
driver_version = super.to_s[/^(\d+\.\d+.\d+)/, 1]
```